packages/babel-parser/src/parse-error/credentials.ts

```diff
@@ -1,10 +1,7 @@
-export const ParseErrorCodes = Object.freeze({
-  SyntaxError: "BABEL_PARSER_SYNTAX_ERROR",
-  SourceTypeModuleError: "BABEL_PARSER_SOURCETYPE_MODULE_REQUIRED",
-});
-
-export type ParseErrorCode =
-  typeof ParseErrorCodes[keyof typeof ParseErrorCodes];
+export enum ParseErrorCode {
```

Comment:
Can you use `const enum` instead? We don't use the reverse mapping anyway.

Reply:
Good catch! Done: https://github.com/babel/babel/pull/14796/commits/c5f4bba84657ec7dbcc8cea7a37ff77cf2be74cc